code/addons/vitest/src/vitest-plugin/index.ts

```diff
@@ -17,14 +18,14 @@ import type {
 } from 'storybook/internal/core-server';
 import { readConfig, vitestTransform } from 'storybook/internal/csf-tools';
 import { MainFileMissingError } from 'storybook/internal/server-errors';
+import { oneWayHash } from 'storybook/internal/telemetry';
```

Comment:
I'm wondering if importing the relative src file would be better?

Reply:
Can you elaborate?

Reply:
Never mind, it's likely nothing.